src/test/java/study/aop/cglibproxy/CglibProxyTest.java

```diff
@@ -0,0 +1,62 @@
+package study.aop.cglibproxy;
+
+import net.sf.cglib.proxy.Enhancer;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.*;
+
+@DisplayName("CGLIB Proxy 테스트")
+public class CglibProxyTest {
+    private final String name = "yang";
+    private HelloTarget proxyInstance;
+
+    @BeforeEach
+    void setUp() {
+        final Enhancer enhancer = new Enhancer();
+        enhancer.setSuperclass(HelloTarget.class);
+        enhancer.setCallback(new CustomMethodInterceptor());
+        proxyInstance = (HelloTarget) enhancer.create();
```

Comment:
```suggestion
        proxyInstance = (HelloTarget) Enhancer.create(HelloTarget.class, new CustomMethodInterceptor());
```
이렇게 한 줄로도 프록시를 적용 할 수 있어요.
아실거라고 생각하지만 혹시 몰라 예시로 코멘트를 남깁니다~

Reply:
정적 메서드를 제공할거란 생각을 못했었네요! 꿀팁 갑사합니다ㅎㅎ